apps/webpack.config.js

```diff
@@ -135,7 +135,7 @@ const nodePolyfillConfig = {
       'process/browser': require.resolve('process/browser'),
       stream: require.resolve('stream-browserify'),
       timers: require.resolve('timers-browserify'),
-      crypto: require.resolve('crypto-browserify'),
+      crypto: false,
```

Comment:
What does this do now?

Reply:
This was introduced when I swapped from crypto-js to crypto-browserify. Since I'm swapping from crypto-browserify to md5, the polyfill is no longer necessary and this reverts to the previous behavior which is not to fallback to a polyfill if crypto cannot be resolved.

Reply:
ah right, forgot this is a semi-revert. Out of curiosity, is this different than this not being here at all? Sounds like this maybe was necesssary in webpack 4, but not 5?

https://webpack.js.org/configuration/resolve/#resolvefallback

Reply:
I believe it is no different than leaving it out (the value would be `undefined`). Webpack 5 dropped automatic polyfills which essentially made them false/undefined by default.

Reply:
I'd prefer to make this change separately so we can unblock storybook, just to make sure removing this statement doesn't cause another breakage. Does that work for you? I can create a followup PR.

Reply:
Fine with me!

Reply:
I'm also in favor of removing any line from the webpack config we don't need. sounds ok to make it its own PR if you'd like.

Reply:
Given how touchy this part of the code has been, I think Stephen's instinct to make it a separate PR is good